data/org.freedesktop.UDisks2.xml

```diff
@@ -2103,11 +2103,14 @@
     <!--
```

Comment:
*Allow resizing LUKS 2 devices* (without *of*)

Reply:
Fixed.